CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+- Support ignoring specific source file pattern when adding them to the target https://github.com/tuist/tuist/pull/811 by @vytis
```

Comment:
will there be any follow up works as i am wondering if the option to exclude files which match certain pattern will be later introduced in the project manifest file?